{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -70,9 +70,9 @@
     "dj_rest_auth",
     "django_filters",
     "django_extensions",
-    {% if cookiecutter.use_graphql == 'y' -%}
+{% if cookiecutter.use_graphql == 'y' -%}
     "graphene_django",
-    {% endif -%}
+{% endif -%}
```

Comment:
yes...I know...slightly gross...but I couldn't figure out a way for it remove lines in a way that wouldn't result in a linting error